test/csharp/edge/Main.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Providers.Null;
+using Providers.Edge;
+using Constructs;
+using HashiCorp.Cdktf;
+
+namespace MyCompany.MyApp
+{
+    // Using references to resource attributes as resource arguments
+    class ReferenceStack : TerraformStack
+    {
```

Comment:
this file is auto generated?

Reply:
No it's hand-written, the autogenerated version does not compile, but was a starting point